File: scala-chapter-3/src/main/scala/org/tlh/springboot/scala/dto/UserDto.scala
package org.tlh.springboot.scala.dto

import scala.beans.BeanProperty


/**
  * <br>
  * Created by hu ping on 6/28/2019
  * <p>
  */
class UserDto extends Serializable {

  @BeanProperty
  var name: String = _

  @BeanProperty
  var age: Int = _


}

File: proguard-chapter-1/src/main/java/org/tlh/springboot/dto/UserDto.java
package org.tlh.springboot.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.io.Serializable;

/**
 * <br>
 * Created by hu ping on 8/15/2019
 * <p>
 */
@Data
public class UserDto implements Serializable {

    private Integer id;
    @JsonProperty(value = "user_name")
    private String name;
    private Integer age;
    private String address;

}
